tests/test_create.py

```diff
@@ -2663,3 +2663,20 @@ def test_create_dry_run_without_prefix(
 
 def test_create_without_prefix_raises_argument_error(conda_cli: CondaCLIFixture):
     conda_cli("create", "--json", "ca-certificates", raises=ArgumentError)
+
+
+def test_nonadmin_file_untouched(
```

Comment:
Should this be a windows only test?

Reply:
The MacOS and Linux installers create the `.nonadmin` file, too, even though it's not propagated into other environments. This is why I am testing this on all platforms.

However, if you think that this test is redundant for non-Windows platforms, I can restrict it to Windows.